pulpcore/tests/functional/api/test_status.py

```diff
@@ -175,3 +175,16 @@ def verify_get_response(status, expected_schema):
     else:
         assert status["storage"]["free"] is not None
         assert status["storage"]["total"] is not None
+
+
+@pytest.mark.parallel
+def test_livez_unauthenticated(
+    pulpcore_bindings,
+    anonymous_user,
+):
+    """
+    Assert that GET requests to Livez API return 200 without a response body.
+    """
+    with anonymous_user:
+        r = pulpcore_bindings.LivezApi.livez_read()
+        assert r is None
```

Comment:
When we expect the reader to understand the meaning of "r" because it must be obvious in the context, we don't even need to give the value a name.

```suggestion
        assert pulpcore_bindings.LivezApi.livez_read() is None
```